enhancements/machine-api/maintenance-lock.md

```diff
@@ -0,0 +1,232 @@
+---
+title: maintenance-lock
+authors:
+  - "@beekhof"
+reviewers:
+  - @bison
+  - @enxebre
+  - @michaelgugino
```

Comment:
can we include people from node team and possibly mco here as well

Reply:
sure